### PR TITLE
Open calendar command

### LINKED ISSUE #140: Calendar window feature
As a Financial Advisor, I want to be see the client's appointment date in a calendar so that I can plan my day better.

### PR DESCRIPTION
**Pull request by syamfarh**
typing `calendar` open the calendar window

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -227,6 +227,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleQuestionnaire();
             }
 
+            if (commandResult.isShowCalendar()) {
```
Comment (Daphne789) [inline src/main/java/seedu/address/ui/MainWindow.java:231]: I think u forgot to add the `calendarWindow.hide();` in the handleExit() method

**Review by Daphne789 — COMMENTED** [review]
Great job, Syam!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (syamfarh) [reply src/main/java/seedu/address/ui/MainWindow.java:231]: Thanks! fixed!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -200,6 +200,7 @@ private void handleExit() {
         logic.setGuiSettings(guiSettings);
         helpWindow.hide();
         questionnaireWindow.hide();
+        calendarWindow.hide();
```
Comment (Daphne789) [inline src/main/java/seedu/address/ui/MainWindow.java:203]: LGTM

**Pull request merged**